robosuite/utils/observables.py

```diff
@@ -9,6 +9,18 @@ def sensor(modality):
 
         any = func(obs_cache)
 
+    An example use case is shown below:
+
+        @sensor(modality="proprio")
+        def joint_pos(obs_cache):
+            # Always handle case if obs_cache is empty
+            if not obs_cache:
```

Comment:
do you mean to check if obs_cache is an empty dictionary, or if it's None? in either case, I prefer to use checks like len(obs_cache) > 0 or obs_cache is not None - there can be tricky bugs by directly checking boolean values

Reply:
Good point -- I think I originally designed the interface to be able to handle either case, though I think the only (current) use case passes in an empty dict